src/Compilers/Core/Portable/CommandLine/CommandLineParser.cs

```diff
@@ -53,13 +56,18 @@ internal virtual TextReader CreateTextFileReader(string fullPath)
         /// <param name="fileNamePattern">File name pattern. May contain wildcards '*' (matches zero or more characters) and '?' (matches any character).</param>
         /// <param name="searchOption">Specifies whether to search the specified <paramref name="directory"/> only, or all its subdirectories as well.</param>
         /// <returns>Sequence of file paths.</returns>
-        internal virtual IEnumerable<string> EnumerateFiles(string directory, string fileNamePattern, SearchOption searchOption)
+        internal virtual IEnumerable<string> EnumerateFiles(string? directory, string fileNamePattern, SearchOption searchOption)
         {
+            if (directory is null)
```

Comment:
What prompted this change?

Reply:
IsAbsolute returns false on `null`, so assertion below fails

---
In reply to: [377820068](https://github.com/dotnet/roslyn/pull/41063#discussion_r377820068) [](ancestors = 377820068)